### DISCUSSION TITLE
[Feature Request] - Filtering out Local Instance from Home View

### DISCUSSION THREAD

**Discussion by AnthonyColletteInSeattle** [mastodon/mastodon#22043] [General]
For the Home view, would it be possible to create a way to filter out the toots from people you follow from the Local timeline, or to be able to filter out people from your own instance in your Home timeline. In the advanced web interface, I'd like to have one column for the Local instance, and another for Home, but I don't want to see local instance activity repeated in the Home column. Thanks!

**Comment by Heusos** [comment]
It would also be helpful to filter out toots from people you already have in a specific list or lists, so that for example you can add a list/column for frequent tooters, and not have them drown out your other follows in the Home column.
Suggestion issue #21388 would cover this.

**Reply by ghost** [reply]
Seconding this suggestion. I want to support news/journalists by following, but I do not want my social media app experience to be one of drowning in news (good or bad). I want to put news in a list and when I have the desire to, I can go look at the list I created but quickly go back to the social side of things when I'm ready.
